tests/foreman/api/test_classparameters.py

```diff
@@ -137,15 +137,15 @@ def test_negative_update_parameter_type(self, test_data, module_puppet):
             2. Error raised for invalid default value.
         """
         sc_param = module_puppet['sc_params'].pop()
+        sc_param.override = True
+        sc_param.parameter_type = test_data['sc_type']
+        sc_param.default_value = test_data['value']
```

Comment:
I guess it's taking out the stuff from the raises block that's not a function call. idk how I feel about this stylistically

Reply:
the point of this change is to make sure that only the call that raises the error should be in the context manager. there are no functional reasons why these extra lines should be under that context.